lib/trusona/workers/paired_tru_code_finder.rb

```diff
@@ -2,6 +2,7 @@
 
 module Trusona
   module Workers
+    # Paired trucode finder
     class PairedTruCodeFinder
```

Comment:
This comment is the same as the class name?

Reply:
I think it's RDoc redundancy